surf-chat/src/main/java/dev/slne/surf/social/chat/listener/PlayerAsyncChatListener.kt

```diff
@@ -115,4 +121,27 @@ class PlayerAsyncChatListener : Listener {
     private fun getCountedPlayers(): Int {
         return Bukkit.getOnlinePlayers().count {!it.hasPermission(SurfChatPermissions.chatLimitBypass) }
     }
+
+    private fun Component.parseItemPlaceholder(player: Player): Component {
+        if (!PlainTextComponentSerializer.plainText().serialize(this).contains("[item]")) {
+            return this
+        }
+
+        val stack = player.inventory.itemInMainHand
+
+        if (stack.type == Material.AIR) {
+            return this
+        }
+
+        return this.replaceText(TextReplacementConfig.builder()
+            .match("[item]")
+            .replacement(when {
+                stack.amount > 1 -> Component.text("${stack.amount}x ", Colors.GOLD).append(stack.displayName())
```

Comment:
Here’s a more concise version using the `text `function from surf-api with the `VARIABLE_VALUE` color:
```suggestion
                stack.amount > 1 -> text("${stack.amount}x ", Colors.VARIABLE_VALUE).append(stack.displayName())
```
Make sure you've imported `text`

Reply:
fixed